### ISSUE TITLE
The inline module shadows vars in the the code module

### ISSUE THREAD

**Issue by michaellenaghan** [imperavi/superkube#19]
Blocks of code are generally placed inside `<pre><code> ... </code></pre>` nested elements.

Given that, the [inline module](https://github.com/imperavi/superkube/blob/main/superkube/modules/inline.scss) declarations like [this](https://github.com/imperavi/superkube/blob/d87dcc75755b4d557b8a9601c657c4ab71a88f88/superkube/modules/inline.scss#L45):

```scss
code {
    ...
    color: var(--inline-code-color);
    ...
}
```

effectively shadow the [code module](https://github.com/imperavi/superkube/blob/c222f2018554a1fa618cadac75f5cb9609675994/superkube/modules/code.scss) declarations like [this](https://github.com/imperavi/superkube/blob/c222f2018554a1fa618cadac75f5cb9609675994/superkube/modules/code.scss#L27):

```scss
pre,
.code {
    ...
    color: var(--code-color);
    ...
}
```

In other words: setting `--code-color` in `pre` or `.code` scope doesn't actually set the code's `color` in a `<pre><code> ... </code></pre>` block.

Until that's resolved, one possible workaround is to set `--inline-code-color` rather than `--code-color`, but in a nested scope (to avoid a possibly unintended global change):

```scss
pre {
    ...
    code {
        --inline-code-color: ...;
    }
}
```
